packages/checkout/src/config.ts

```diff
@@ -0,0 +1,4 @@
+export const CHECKOUT_ENDPOINT_SBX = 'https://sbx.fung.money';
```

Comment:
Could we also have our DEV endpoint as an option? Merchants don't have API keys there so they could never use it

Reply:
@ashwini-fung